cmd/nvidia-container-runtime/main_test.go

```diff
@@ -105,8 +105,8 @@ func TestGoodInput(t *testing.T) {
 		t.Fatalf("error generating runtime spec: %v", err)
 	}
 
-	//nolint:gosec // TODO: Can we harden this so that there is less risk of command injection
-	cmdRun := exec.Command(nvidiaRuntime, "run", "--bundle", cfg.bundlePath(), "testcontainer")
+	//nolint:gosec
+	cmdRun := exec.Command(oci.EscapeArg(nvidiaRuntime), oci.Escape([]string{"run", "--bundle", cfg.bundlePath(), "testcontainer"})...)
```

Comment:
```suggestion
	cmdRun := exec.Command(oci.Escape(nvidiaRuntime)[0], oci.Escape([]string{"run", "--bundle", cfg.bundlePath(), "testcontainer"})...)
```

This way we could have `EscapeArg` private

Reply:
That works too! Reverted